ibis/expr/types/generic.py

```diff
@@ -2684,6 +2684,24 @@ def nth(self, n: int | ir.IntegerValue) -> Column:
         """
         return ops.NthValue(self, n).to_expr()
 
+    def to_list(self, **kwargs) -> list:
+        """Convert a column expression to a list.
+
+        Parameters
+        ----------
+        kwargs
+            Same as keyword arguments to [`to_pyarrow`](#ibis.expr.types.core.Expr.to_pyarrow)
+
+        Examples
+        --------
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> t = ibis.examples.penguins.fetch().limit(5)
+        >>> t.bill_length_mm.to_list()
```

Comment:
Do you want to move the limit into the `to_list` call here to demonstrate that option?

Reply:
Done!